.github/workflows/gatk-nightly-docker.yml

```diff
@@ -1,10 +1,9 @@
 name: Build and Upload GATK Nightly Docker Image
 on:
   schedule:
-    # trigger the job the image at 1am est every night (UTC 20:00)
-    # NOTE: by default schedule:cron jobs pull from the repo default branch which makes this the master branch of gatk
-    - cron: '0 20 * * *'
+    - cron: '17 05 * * *' # trigger the job the image at 1:17am est every night (UTC 05:17)
```

Comment:
It's actually 1:17 AM EDT (Eastern Daylight Time), but 12:17 AM EST (Eastern Standard Time). Can you update the comment?

Reply:
good point. I don't know how time zones work